app/modules/toolkits/rdkit_wrapper.py

```diff
@@ -160,8 +160,12 @@ def get_tanimoto_similarity_rdkit(
     if mol1 and mol2:
         if fingerprinter == "ECFP":
             # Generate Morgan fingerprints for each molecule
-            fp1 = AllChem.GetMorganFingerprintAsBitVect(mol1, radius, nBits)
-            fp2 = AllChem.GetMorganFingerprintAsBitVect(mol2, radius, nBits)
+            fp1 = AllChem.GetMorganFingerprintAsBitVect(
+                mol1, int(radius / 2), nBits, useChirality=True
```

Comment:
The wording is the wrong way round. The 2/4/6 in ECFP2/4/6 denote the diameter. The RDKit fingerprinter interprets the given number as the radius. So if you say in the doc that the param is interpreted as a radius, you should double it for the ECFP generation instead of halving it. If you say in the doc that the param is interpreted as a diameter (hypothetically), you should halve it for the RDKit FP generation.

Add default values for radius and nBits in doc.